asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/dummy/AbstractWrapper.kt

```diff
@@ -432,7 +434,13 @@ abstract class AbstractWrapper(val publisher : PulsarPublisher, val wpsConfigura
         // Now, we know that we haven't found any existing link for it.
         // so we are going to upload it.
         val fileStorage = FileStorage(filestorageConfig.endpoint, filestorageConfig.user, filestorageConfig.password)
-        fileStorage.upload(filestorageConfig.bucketName, checksum, content, mimeType)
+        retry(wpsConfiguration.retryConfiguration.maxRetries, wpsConfiguration.retryConfiguration.backoffMillis, { ex ->
```

Comment:
one could argue if it should still be WPS retry configuration if we use it for upload (and download)

Reply:
Yes, you are right.

Either we make it more general, or we put use the retry setting for 2 different broader configurations (one for the wps, one for the filestorage).

Currently I tend a little bit towards two different sub configurations.